subsys/bootloader/CMakeLists.txt

```diff
@@ -13,3 +13,7 @@ add_subdirectory_ifdef(CONFIG_SECURE_BOOT_DEBUG debug)
 
 include(${CMAKE_CURRENT_LIST_DIR}/cmake/fw_info_magic.cmake)
 zephyr_include_directories(include)
+
+# This is added to create a fake b0_kernel_elf target which can be
+# dependend on.
+set_property(GLOBAL APPEND PROPERTY ${IMAGE}GENERATED_KERNEL_SOURCE_FILES ${ZEPHYR_BASE}/misc/empty_file.c)
```

Comment:
This need some attention from the reviewers. I'm not sure if this is the best way of doing what the comment above explains.

Reply:
Where is b0_kernel_elf used?

Reply:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/620/files#diff-fff29384b321a9c75585f016d50e5db1R16

Reply:
It's basically a phony_target to make sure that b0_kernel_elf and kernel_elf is merged.

Reply:
So if you have something that needs to be merged into the b0_kernel_elf before b0_kernel_elf and kernel_elf is merged you could append it to the dependencies of b0_kernel_elf. at least that's the idea.

Reply:
Removed it @SebastianBoe found a better way to do it. Now it relies on `logical_target_zephyr_elf` .